website/docs/docs/on-premises-2x/environment-variables.md

```diff
@@ -166,6 +166,13 @@ Required for running Postgres on RDS with IAM authentication. This must be set t
 
 **Required:** false<br/>
 
+### REDIS_URL
+
+The fully qualified connection string to a RESP-compatible service. Only the `redis://` protocol is supported.
```

Comment:
Do we not support `valkey://` as well?

Reply:
Also, what about the TLS variants: `valkeys://` and `rediss://`?

And if we do support TLS variants, do we accept self-signed certificates?

Reply:
https://github.com/redis/ioredis?tab=readme-ov-file#connect-to-redis Ie `redis://` or `rediss://`

We're actually using valkey ourselves, and using `redis://` works with that